### PR TITLE
Add fallback when ScratchDB forgets to `res.send()`

### LINKED ISSUE #50: Searching "null" doesnt work
https://scratchinfo.vercel.app/users/null  --- doesn't show anything. Stuck on loading page .

**To Reproduce**

1. Go to https://scratchinfo.vercel.app/users/null
2. Wait for 10 years. Then you know it will never load lol

**Expected behavior**
"null" user exists on scratch. So I think It should load.

**Smartphone (please complete the following information):**

- Device:  Samsung tab 
- OS: Android
- Browser Chrome

**Additional context**
null (lol)

### PR DESCRIPTION
**Pull request by hello-smile6**
Resolves #50 
(maybe)

- [x] If this checkbox isn't checked, I'm not finished yet. I need to change something.

### CODE REVIEW

File: src/routes/api/[user].ts [hunk 1]
```diff
@@ -8,7 +8,22 @@ export async function get({ params }) {
...
+          {"date":"1234-12-23T12:34:56.000Z","value":1},
+        ],
+        "firstSeen":{
+          "date":"1234-12-23T12:34:56.000Z","id":1
+        },
+        "lastSeen":{
+          "date":"1234-12-23T12:34:56.000Z","id":1
+        },
+        "signature":"Well, that one fell off the database. ScratchDB didn't know what to say.",
+      };
```
Comment (hello-smile6) [inline src/routes/api/[user].ts:26]: Copied the data from my profile.

Comment (webdev03) [reply src/routes/api/[user].ts:26]: Hey, you should turn it into a Typescript interface!

Comment (hello-smile6) [reply src/routes/api/[user].ts:26]: > Hey, you should turn it into a Typescript interface!

How?

File: src/routes/api/[user].ts [hunk 2]
```diff
@@ -25,9 +25,19 @@ export async function get({ params }) {
...
+      // Suspect scratchDB used == instead of ===
+    }
+    else {
+      try {
+      scratchdb_forum_user = await (
+        await fetch(`https://scratchdb.lefty.one/v3/forum/user/info/${user}`)
+      ).json();
+      }
+      catch(e) {
+        scratchdb_forum_user.signature="Well, ScratchDB didn't know. So we don't know."
```
[the remark this answers is not in the document]
Comment (hello-smile6) [reply src/routes/api/[user].ts:38]: > 🤣

Had to be silly.

File: src/routes/api/[user].ts (hunk 1 above)
Comment (webdev03) [reply src/routes/api/[user].ts:26]: Example:

``ts
interface Person {
// name is type string
  name: string;
// age is type number
  age: number;
// next is optional
// type[] means it in array
  friends?: Person[];
}
```ts

Comment (hello-smile6) [reply src/routes/api/[user].ts:26]: > Example:
> 
> ```ts
> interface Person {
> // name is type string
>   name: string;
> // age is type number
>   age: number;
> // next is optional
> // type[] means it in array
>   friends?: Person[];
> }
> ```

Okay???

Comment (webdev03) [reply src/routes/api/[user].ts:26]: Nevermind

**Comment by webdev03** [comment]
Why doesn't this go on the client side?

**Comment by hello-smile6** [comment]
> Why doesn't this go on the client side?

Because the server already has logic for a fallback. Additionally, the users' browser will most likely take longer to connect than the server. By doing it server-side, pages will load faster.

**Comment by webdev03** [comment]
Wait I have an idea! I'm going to make it load from a load function so that means it works without JS!

**Pull request closed without merging**